<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorGUI</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_div">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_mul">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_min">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_equal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_add">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_dot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_sym">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
